utils/pciutils/Makefile

```diff
@@ -54,20 +63,27 @@ define Build/Prepare
 	$(CP) $(DL_DIR)/$(PCI_IDS_FILE) $(PKG_BUILD_DIR)/pci.ids
 endef
 
+TARGET_CFLAGS += -fPIC
+
 MAKE_FLAGS += \
 	CFLAGS="$(TARGET_CFLAGS) $(TARGET_CPPFLAGS)" \
 	PREFIX="/usr" \
 	HOST="Linux" \
 	HWDB="no" \
-	ZLIB="yes"
+	ZLIB="yes" \
+	SHARED="yes"
 
 ifneq ($(CONFIG_USE_GLIBC),)
 TARGET_LDFLAGS += -lresolv
 endif
 
 define Build/InstallDev
 	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_BUILD_DIR)/lib/libpci.a $(1)/usr/lib
+	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libpci.so.3 \
```

Comment:
This looks odd given the command below, sure you don't want to create links?

Reply:
the source already is a symlink, but it must be this one too otherwise ti will complain of missing -lpci